texastribune/donations
# Improve error handling and give Sentry less user information

#### What's this PR do?
Makes it so we give less about the request/response and a logged-in user's information to Sentry.

#### Why are we doing this? How does it help us?
It's always good store as little information about users as possible, especially in third-party services.

#### How should this be manually tested?
Ensure you have this in your `env-docker`:
```
ENABLE_SENTRY=True
SENTRY_DSN=https://d776f8ab57804e8a8d0f581ed0e4d3b2@sentry.io/1304832
SENTRY_ENVIRONMENT=<your name>
```
Then:
- `make`
- `make restart`
- Go to `/account/` and enter `bleh` in the fixed-position "view as" box on the left
- You should get an error page, and if you go to Sentry under your environment, there should be a new issue. Confirm the only info included under USER is IP address and user ID.

#### How should this change be communicated to end users?
NA.

#### Are there any smells or added technical debt to note?
NA.

#### What are the relevant tickets?
https://3.basecamp.com/3098728/buckets/736178/todos/2638868264

#### Have you done the following, if applicable:
